src/optimizer/query_graph.c

```diff
@@ -8699,6 +8704,14 @@ qo_discover_sort_limit_nodes (QO_ENV * env)
   return;
 
 abandon_stop_limit:
+  /* remove error which is occured in this function */
+  if (pt_has_error (QO_ENV_PARSER (env)))
+    {
+      pt_reset_error (QO_ENV_PARSER (env));
+      (QO_ENV_PARSER (env))->flag.has_internal_error = 0;
+    }
+
```

Comment:
On lines 8617 and 8626, is not bitset_delset() necessary for expr_segs and tmp_bitset variables?

Reply:
no. Since it is not allocated, bitset_delset() is not needed.